doc/source/programs/gdal_vector_index.rst

```diff
@@ -0,0 +1,167 @@
+
+    The output field name to hold the file path/location to the indexed
+    vectors. The default field name is ``location``.
+
+.. option:: --absolute-path
+
+    The absolute path to the vector files is stored in the index file.
+    By default the vector filenames will be put in the file exactly as they
+    are specified on the command line.
+
+.. option:: --dst-crs <DST-CRS>
+
+    Geometries of input files will be transformed to the desired target
+    coordinate reference system.
+    Default creates simple rectangular polygons in the same coordinate reference
+    system as the input vectors.
+
+.. option:: --source-crs-field-name <SOURCE-CRS-FIELD-NAME>
+
+    The name of the field to store the CRS of each tile. This field name can be
+    used as the value of the TILESRS keyword in MapServer
+
+.. option:: --source-crs-format auto|WKT|EPSG|PROJ
+
+    The format in which the CRS of each tile must be written. Types can be
+    ``auto``, ``WKT``, ``EPSG``, ``PROJ``.
+    This option should be used together with :option:`--source-crs-field-name`.
+
+.. option:: --source-layer-name <SOURCE-LAYER-NAME>
+
+    Add layer of specified name from each source file in the tile index [may be repeated]
+    If none of
+
+.. option::  --source-layer-index <SOURCE-LAYER-INDEX>
+
+    Add layer of specified index (0-based) from each source file in the tile index [may be repeated]
+
+.. option:: --metadata <KEY>=<VALUE>
+
+    Write an arbitrary layer metadata item, for formats that support layer
+    metadata.
+    This option may be repeated.
+
+.. option:: --accept-different-crs
+
+    Whether layers with different CRS are accepted.
+    By default, unless :option:`--dst-crs` is specified, layers that do not have
+    the same CRS as the first index layer will be skipped. Setting :option:`--accept-different-crs`
+    may be useful to avoid the CRS consistency check.
+
+.. option:: --accept-different-schemas
+
+    Whether layers with different schemas are accepted.
+    By default, layers that do not have the same schemas as the first index layer
+    will be skipped.
+
+.. option:: --dataset-name-only
+
+    Whether to write the dataset name only, instead of suffixed with the layer index.
+
+    .. warning:: Setting this option will generate a location not compatible of MapServer.
+
+
+Advanced options
+++++++++++++++++
+
+.. include:: gdal_options/oo.rst
+
+.. include:: gdal_options/if.rst
+
+Examples
+--------
+
+.. example::
+
+   Produce a GeoPackage with a record for every
+   layer that the utility found in the ``countries`` folder. Each record holds
+   information that points to the location of the image and also a bounding rectangle
+   shape showing the bounds of the image:
+
+   .. code-block:: bash
+
+      gdal vector index countries/*.shp index.gpkg
+
+.. example::
+
+   The :option:`--dst-crs` option can also be used to transform all input vector
+   geometries into the same output projection:
```

Comment:
Same as above, maybe "envelopes" because "vector geometries" may be misleading.

Reply:
fixed